GVRf/Framework/src/org/gearvrf/GVRAssimpImporter.java

```diff
@@ -31,7 +33,15 @@
     int getNumberOfMeshes() {
         return NativeAssimpImporter.getNumberOfMeshes(getPtr());
     }
-
+    
+    /**
+     * @return Loads the 3D model.
+     */
+    GVRScene loadScene(Bitmap bitmap)
```

Comment:
Shouldn't this method be returning a `GVRSceneObject`, not a `GVRScene`? Scenes contain scene objects, which may in turn contain scene objects. Most scenes will be composed of multiple scene objec ts, loaded from multiple model files.